idl.cmake

```diff
@@ -135,8 +135,14 @@ ENDMACRO(GENERATE_IDL_CPP FILENAME DIRECTORY)
 #                    Can be prefixed by a path: _path/_filename
 #   :param DIRECTORY: IDL directory.
 #                     The idl file being search for is: ``${DIRECTORY}/${_filename}.idl``
+#   :param ARGUMENTS:  The following words are passed as arguments to omniidl
 #
 MACRO(GENERATE_IDL_PYTHON FILENAME DIRECTORY)
+  SET(options)
+  SET(oneValueArgs )
+  SET(multiValueArgs ARGUMENTS)
```

Comment:
Where the `ARGUMENTS` comes from here?

Reply:
It tells CMake to parse this in the remaining arguments, i.e. `foo (a b ARGUMENTS c d)` leads to `_omni_ARGUMENTS` contain `c;d`

Reply:
OK, thanks for this clarification.